_ingest-pipelines/processors/index-processors.md

```diff
@@ -69,6 +69,12 @@ Processor type | Description
 `urldecode` | Decodes a string from URL-encoded format.
 `user_agent` | Extracts details from the user agent sent by a browser to its web requests. 
 
+## Validations on Processors
+
+We can configure the limits on the number of ingest processors that should be used. The limit can be configured using the attribute `cluster.ingest.max_number_processors`. The sum of the number of processors and the number of `on_failure` processors are considered for counting the total number of processors on which the limit would be applied.
```

Comment:
> The limit can be configured using the attribute

The limit can be configured using the cluster setting

Reply:
Updated